lang/src/org/partiql/lang/CompilerPipeline.kt

```diff
@@ -273,7 +285,8 @@ internal class CompilerPipelineImpl(
             }
         }.flatten().toMap(),
         procedures,
-        compileOptions
+        compileOptions,
+        aliasesToScalarType
```

Comment:
I suggest not using raw types like Map and List. Wrapping types like this in simple holder classes introduces indirection which is helpful for future changes

Reply:
Will apply this comment